### PR TITLE
Move guess_anchor_molecules to Topology

### LINKED ISSUE #1197: Move guess anchor atoms to topology

### PR DESCRIPTION
**Pull request by mpharrigan**
fixes #1197

### CODE REVIEW

File: mdtraj/core/trajectory.py [hunk 1]
```diff
@@ -1857,32 +1857,6 @@ def _check_valid_unitcell(self):
     def _have_unitcell(self):
         return self._unitcell_lengths is not None and self._unitcell_angles is not None
 
-    def guess_anchor_molecules(self):
```
Comment (marscher) [inline mdtraj/core/trajectory.py]: you could deprecate the method first and redirect it to the topology impl.

Comment (mpharrigan) [inline mdtraj/core/trajectory.py]: It was never in a release. This function was added a couple of weeks ago and I said it should be on Topology, but the move wasn't performed at the time

**Comment by marscher** [comment]
Ah okay ;)

File: mdtraj/core/topology.py [hunk 2]
```diff
@@ -1111,6 +1111,30 @@ def find_molecules(self):
             molecules[atom_molecule[atom.index]].add(atom)
         return molecules
 
+    def guess_anchor_molecules(self):
+        """Guess anchor molecules for imaging
```
Comment (peastman) [inline mdtraj/core/topology.py]: If I didn't already know, I'd be confused about what that description was talking about.  Perhaps add a "see also" referencing `Trajectory.image_molecules()`?

Comment (mpharrigan) [inline mdtraj/core/topology.py]: Good point

**Pull request merged**